instance/jmsml_D_Control_Measure.xml

```diff
@@ -3664,7 +3664,6 @@
             <DigitTwo>8</DigitTwo>
           </EntityTypeCode>
         </EntityType>
-        <EntityType ID="TMT_DESTROY" Label="Destroy" Graphic="25340900.svg">
```

Comment:
@Dbarnes1 , I should bounce this PR back to you because this line got accidentally deleted from your commit, but in the interest of getting this done before today's sprint review and planning, I'm going to merge this PR then restore this missing line.  Please make sure you update your personal and local version of this file also, to restore this missing line in your own XML.